docs/content/tutorial/step_03.ngdoc

```diff
@@ -88,6 +88,19 @@ Let's see an example:
     });
 ```
 
+```html
+<html ng-app="myApp">
+<head>
+  <script src="bower_components/angular/angular.js"></script>
+  <script src="app.js"></script>
+</head>
+<body>
+   <!-- The following line is how to use the greetUser angular.js controller/directive above in your html doc-->
```

Comment:
I would rather you used the word `component` here rather than `controller/directive`.

Reply:
Thanks, updated the wording